README.md

```diff
@@ -5,12 +5,31 @@ This repository contains bash and python scripts for the the release automation
 ## Usage
 
 It is assumed below that the release script is run from the root directory of the repository.
+It is also assumed that Miniconda or Anaconda is installed on your Linux distribution.
 
 To display the usage, use
 
 ```bash
 ./scripts/automation/release.sh --help
 ```
+The file `./scripts/globals.sh` contains global variables used across all scripts. To work with test branches 
```

Comment:
The forked repositories must be removed, particularly private ones, and so are the associated TeamCity sub-projects. They clutter the parent project. It does not make sense to leave there forever. They were there for development purposes. 

Also notice that that the jobs that generate nupkgs from the forks, have the exact same name prefixes as the real nupkgs, and end up in the package repository. That's bad. They are visible in visual studio through the nuget manager.